app/src/main/java/com/weatherxm/ui/common/UIModels.kt

```diff
@@ -453,3 +467,16 @@ enum class AnnotationCode : Parcelable {
     UNIDENTIFIED_ANOMALOUS_CHANGE,
     UNKNOWN
 }
+
+@Parcelize
+enum class AnnotationGroupCode : Parcelable {
+    NO_WALLET,
+    NO_STATION_DATA,
+    SENSOR_PROBLEMS,
+    WEATHER_DATA_GAPS,
+    BAD_STATION_DEPLOYMENT,
+    NO_LOCATION_DATA,
+    LOCATION_NOT_VERIFIED,
+    USER_RELOCATION_PENALTY,
+    UNKNOWN
```

Comment:
I'm not sure if the client should know about the other cases since the logic is to make the client agnostic of annotations. 
IMO we should handle only the `NO_WALLET ` and `LOCATION_NOT_VERIFIED` cases

Reply:
I'll agree with @pantelisss on this. I can see a potential use of the others in the future, but it's not part of this feature and they could change names again.